nss-c31-mixed-havoc/nutshell
# Cm bootstrap

# Description

I fixed a few bugs on the events section. Still need to fix the cancel button. Also created the functionality for the registration form. You shouldn't be able to register with a taken username/email. When you register a new user, it should load the page. The register should also post the new user to the database.

Applies to issue # Events / Registration

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. `git fetch --all`
2. `git checkout cm-bootstrap`
3. `cd src/lib`
4. Run grunt
5. http://localhost:8080
6. Try to register an account with your email and username
7. Make sure it doesn't allow you to register
8. If a new user registers, make sure the load function works.
9. Check database to make sure new user is posted.

# Checklist:
